src/viur/core/bones/base.py

```diff
@@ -1212,8 +1212,21 @@ def _compute(self, skel: 'viur.core.skeleton.SkeletonInstance', bone_name: str):
 
         ret = self.compute.fn(**compute_fn_args)
 
+        def unserialize_raw_value(value: list[dict] | dict | None):
+            if self.multiple:
+                return [self.singleValueUnserialize(value)
+                        for value in ret]
+            else:
+                return self.singleValueUnserialize(ret)
```

Comment:
```suggestion
            return self.singleValueUnserialize(ret)
```

Reply:
To quote the second zen of python:
> Explicit is better than implicit.

But if the maintainer wants it that way, that's fine for me. :smiley:

Reply:
Keep it short and simple. <3